src/components/subscriptions/edit/AddSubAddon.js

```diff
@@ -0,0 +1,127 @@
+import React, { useState } from "react";
+import { useMutation, useQueryClient } from "react-query";
+
+import { Field, Form, Formik } from "formik";
+import { Button, MenuItem, Typography } from "@material-ui/core";
+import DEDialog from "components/utils/DEDialog";
+import FormTextField from "components/forms/FormTextField";
+import { announce } from "components/announcer/CyVerseAnnouncer";
+import ErrorTypographyWithDialog from "components/error/ErrorTypographyWithDialog";
+
+import ids from "../ids";
+import buildID from "components/utils/DebugIDUtil";
+import { useTranslation } from "next-i18next";
```

Comment:
```suggestion
import { useTranslation } from "i18n";
```

Reply:
I believe I was letting VSCode autofill this import for me, thank you for catching this.